docs/modules/tls.md

```diff
@@ -252,16 +252,16 @@ none
 
 ## tls.cert.verify()
 
-Controls the vertificate verification process when the Nodemcu makes a secure connection.
+Controls the certificate verification process when the Nodemcu makes a secure connection.
 
 #### Syntax
 `tls.cert.verify(enable)`
 
-`tls.cert.verify(pemdata)`
+`tls.cert.verify(pemdata[, pemdata])`
```

Comment:
Please consider updating the comment at https://github.com/nodemcu/nodemcu-firmware/blob/1c83f027e2e4972c66a5dd2db71d87ab3ae80a15/app/modules/tls.c#L568 as well to be less confusing.  The two signatures given here should be copied over.